HotSpot/Models/Booking.h

```diff
@@ -7,15 +7,17 @@
 //
 
 #import <Parse/Parse.h>
+#import "TimeInterval.h"
 #import "Listing.h"
 
 NS_ASSUME_NONNULL_BEGIN
-
+@class Listing;
```

Comment:
if you have the @class, you don't need to import Listing.h in the header file. this is called a forward declaration (read up about it).

Reply:
and if removing Listing.h works, you can forward declare TimeInterval too